hr_timesheet_task_stage/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2016 Tecnativa - Sergio Teruel
+# Copyright 2016-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2018 Tecnativa - Ernesto Tejeda
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# Copyright 2020 CorporateHub (https://corporatehub.eu)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Task Log: Open/Close Task",
+    "version": "14.0.1.1.0",
+    "category": "Operations/Timesheets",
+    "website": "https://github.com/OCA/timesheet",
+    "author": "Tecnativa, " "CorporateHub, " "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "application": False,
+    "summary": "Open/Close task from corresponding Task Log entry",
+    "depends": ["hr_timesheet", "project_stage_closed"],
```

Comment:
In v14 we dont need the module "project_stage_closed" in dependency in base we already have is_closed field for "project.task.type" model